### ISSUE TITLE
more description for download files

### ISSUE THREAD

**Issue by Siri101** [eggnogdb/eggnog_docs#9]
As shown in https://github.com/eggnogdb/eggnog_docs/wiki/Description-of-download-files, most download files have corresponding descriptions, but some files like e6.func_profiles.json and e6.seq2ogs.tsv have no descriptions. Looking forward a more detailed descriptions.

**Comment by Cantalapiedra** [comment]
Thank you!

**Comment by yudalang3** [comment]
Yes, need more detailed doc. As I spend a lot of time to know, How to recover the evolutionary history of a target gene!
